andela/temari-cfh
# #137091013 login signup page

#### What does this PR do?
This PR adds login and sign up page

#### How should this be manually tested?
The user should fill the sign up and sign in form, click the buttons and be signed in.

#### What are the relevant pivotal tracker stories?
As a user i should see a redesigned(according to material design specs and chosen color scheme) login/signup screen with the option to authenticate via facebook, twitter and google.

Acceptance Criteria
GIVEN a user is directed to an authentication page
WHEN the user tries to login/signup
THEN they should see a redesigned interface

DEV NOTES
This should be loaded from the server

DESIGN Notes
Design the page to be in line with chosen brand colors. A mockup should first be designed before it is implemented.
